### DISCUSSION TITLE
Exracting without macros?

### DISCUSSION THREAD

**Discussion by HommeSauvage** [lingui/js-lingui#1191] [Q&A]
I am using NextJS and I want to keep using SWC which is not compatible with macros yet. I was wondering if there's a way to let Lingui extract messages without me using macros, just i18n instance and <Trans> from @lingui/react.

**Comment by timofei-iatsenko** [comment]
It should work out of the box.

**Comment by Erij-Maherzia-BEN-BRAHIM** [comment]
same thing using Remix

**Comment by timofei-iatsenko** [comment]
Did you try to extract without macros? Macro is opt-in, extracting should work with no issue.
